src/pages/RunDetails.test.js

```diff
@@ -221,4 +245,38 @@ describe('RunDetails - render smoke test', () => {
       expect(wdlScript).toBeDefined
     })
   })
+
+  it('shows the execution log button', async () => {
+    render(h(RunDetails, runDetailsProps))
+    await waitFor(() => {
+      const executionLog = screen.getByText('Execution log')
+      expect(executionLog).toBeDefined
+    })
+  })
+
+  it('correctly identifies azure URIs', () => {
+    expect(isAzureUri('https://coaexternalstorage.blob.core.windows.net/cromwell/user-inputs/inputFile.txt'))
```

Comment:
One nitpick: UI tests like to use matchers after `expect` since it explicitly states the test outcome (both in the code as well as the test output should the test fail). Matchers are basically the `toBe(<expected value>)` or `toBeDefined` calls chained onto the `expect` statements on the tests.

Jest docs have a list of matchers (with usage examples), making it a go to reference point.